helper/colorlogger.py

```diff
@@ -14,7 +18,7 @@ def create_logger(name, color='reset', log_level=logging.INFO, log_colors={
     handler.setFormatter(colorlog.ColoredFormatter('%(log_color)s%(asctime)s %(' + color + ')s[%(module)10s] %(log_color)s[%(levelname)5s] %(' + color + ')s%(message)s',
                                                    log_colors=log_colors))
     log = colorlog.getLogger(name)
-    log.propagate = False
+    log.propagate = True
```

Comment:
because it's easier to let it propagate to the root scope and add a file handler to the root scope instead of setting the file handler for every scope individually